src/SME.SGP.Aplicacao/Commands/EncaminhamentoAEE/Pendencias/GerarPendenciaCPEncaminhamentoAEE/GerarPendenciaCPEncaminhamentoAEECommandHandler.cs

```diff
@@ -85,5 +86,16 @@ public async Task<bool> Handle(GerarPendenciaCPEncaminhamentoAEECommand request,
             }
             return false;
         }
+        private async Task ExcluirPendenciasEncaminhamentoAEE(long encaminhamentoId)
+        {
+            var pendenciasEncaminhamentoAEE = await mediator.Send(new ObterPendenciasDoEncaminhamentoAEEPorIdQuery(encaminhamentoId));
+            if (pendenciasEncaminhamentoAEE != null || !pendenciasEncaminhamentoAEE.Any())
```

Comment:
aqui não seria "if (pendenciasEncaminhamentoAEE != null && pendenciasEncaminhamentoAEE.Any())"? pois a negação do .Any() entraria dentro desse if, se caso a lista de pendências for vazia